### DISCUSSION TITLE
No response on support ticket #3076135

### DISCUSSION THREAD

**Discussion by DavidWarburton-gov-bc** [community/community#145587] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I made a ticket trying to regain access to an account 3 weeks ago and have got no response. I saw at least one other person succeed in getting a response for their ticket by posting to this discussion, so I'm trying that. Is there anything else I'm supposed to do to get a response?

**Comment by ujamX** [comment, marked as answer]
I also remember deleting my account and getting it restored.
I remember creating several tickets because there was no response.
The method that was effective for me was to write a post every day to share the current progress in a ticket I had already created.
That's how I got a reply and the deleted account was restored according to the procedure.
I wish you good results

**Reply by techydevi** [reply]
What the??? So they won't respond unless you have multiple tickets? That's tedious and not...good. Thanks for helping.

**Reply by ujamX** [reply]
I didn't mean that I wouldn't respond if I didn't have several copies of the ticket.
However, I also wanted to restore my account so much that I created several tickets and I remember replying to each ticket asking to be informed of the progress...
I hope my method helps.

**Reply by DavidWarburton-gov-bc** [reply]
Thanks for your advice. From what I've seen, it seems like the main thing that gets a response is persistence. I have so far not engaged with this process very actively, I just left my ticket and waited to be contacted. Even this post, I wrote it and then I went back to my other work, and didn't engage with it in my off time.

Starting today, I'll make sure to comment on my ticket every day and see if that works. Thanks again!
